activesupport/lib/active_support/multibyte/unicode.rb

```diff
@@ -138,21 +138,6 @@ def pack_graphemes(unpacked)
         unpacked.flatten.pack("U*")
       end
 
-      # Re-order codepoints so the string becomes canonical.
-      def reorder_characters(codepoints)
```

Comment:
This method is public but it seems created for internal use.
Can't we remove this method or should emit deprecation warn?